specification/quota/resource-manager/Microsoft.Quota/preview/2019-07-19-preview/quota.json

```diff
@@ -0,0 +1,591 @@
+        "operationId": "QuotaRequests_Create",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ProviderIdParameter"
+          },
+          {
+            "$ref": "#/parameters/LocationIdParameter"
+          },
+          {
+            "$ref": "#/parameters/QuotaRequestNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "createQuotaRequests",
+            "in": "body",
+            "description": "Quota requests payload.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CreateGenericQuotaRequestParameters"
+            }
+          }
+        ],
+        "responses": {
+		 "200": {
+            "description": "OK"
+          },
+          "202": {
+            "description": "Accepted. Quota requests will be created asynchronously. Please use requestName to check the status."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ExceptionResponse"
+            }
+          }
+        },
+		"x-ms-long-running-operation": true
+      },
+      "get": {
+        "tags": [
+          "QuotaRequests"
+        ],
+        "summary": "For the specified location, gets the Quota request status by request name provided during the quota request creation for the specific resources under the subscription.",
+        "description": "Gets the QuotaRequest details and status by the quota Request Name for the resources for the resource  provider at a specific location.",
+        "operationId": "QuotaRequests_Get",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ProviderIdParameter"
+          },
+          {
+            "$ref": "#/parameters/LocationIdParameter"
+          },
+          {
+            "$ref": "#/parameters/QuotaRequestNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok. Returning the status for all relevant quota requests .",
+            "schema": {
+              "$ref": "#/definitions/QuotaRequestDetailsByName"
```

Comment:
The schema for put input should be the same as the get response. In this case: CreateGenericQuotaRequestParameters.

Can these types be combined?

Reply:
The reason explained above.
The put /quotaRequests is constructed from get /usages . They are the pair for get and put.

Reply:
Updated the API to match Get /serviceLimits and Put. Used the same data model.

Reply:
The wrapper is changed. The get response has  "nextLink": "string". 
The Put shouldn't have  "nextLink": "string".
Any suggestions?

Reply:
Put doesn't have nextLink, as the whole set of subrequests is treated as single request.